src/Azure/Orleans.Persistence.AzureStorage/Providers/Storage/AzureTableStorageOptions.cs

```diff
@@ -45,6 +45,27 @@ public class AzureTableStorageOptions
         public TypeNameHandling? TypeNameHandling { get; set; }
         #endregion json serialization
     }
+    /// <summary>
+    /// Configuration validator for AzureTableStorageOptions
+    /// </summary>
+    public class AzureTableGrainStorageOptionsValidator : IConfigurationValidator
+    {
+        private readonly AzureTableStorageOptions options;
+        private readonly string name;
+        public AzureTableGrainStorageOptionsValidator(AzureTableStorageOptions options, string name)
+        {
+            this.options = options;
+            this.name = name;
+        }
+
+        public void ValidateConfiguration()
+        {
+            if (string.IsNullOrEmpty(this.options.ConnectionString))
```

Comment:
Should we parse it to check that it is in fact an azure connection string? A simple call to `CloudStorageAccount.TryParse`maybe?

Reply:
good idea

Reply:
I think we should also validate that the `TableName` is valid too